.gitignore

```diff
@@ -4,3 +4,5 @@ build
 /doc/temp/
 /docs/
 /tools/phpunit
+*.swo
```

Comment:
Editor specific files do not belong in the project's git ignore file, they belong in a developers local [global git ignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).